starter-code/js/index.js

```diff
@@ -1,52 +1,170 @@
-function deleteItem(e){
+// qty
 
+function getQty() {
+    var qty = document.querySelector("#qty").value;
+    document.querySelector("#total").innerHTML = qty;
+    return qty;
 }
 
-function getPriceByProduct(itemNode){
+getQty();
 
-}
 
-function updatePriceByProduct(productPrice, index){
+// qty2
 
+function getQty2() {
+    var qty2 = document.querySelector("#qty2").value;
+    document.querySelector("#total2").innerHTML = qty2;
+    return qty2;
 }
 
-function getTotalPrice() {
+getQty();
 
-}
 
-function createQuantityInput(){
+// Price
 
+function getPrice() {
+    var price = parseInt(document.querySelector("#product").innerHTML);
+    document.querySelector("#total").innerHTML = price * getQty();
 }
 
-function createDeleteButton(){
+getPrice();
+
+// Price 2
 
+function getPrice2() {
+    var price2 = parseInt(document.querySelector("#product2").innerHTML);
+    document.querySelector("#total2").innerHTML = price2 * getQty2();
 }
 
-function createQuantityNode(){
+getPrice2();
 
-}
+// get elements by class name then do a loop
 
-function createItemNode(dataType, itemData){
 
+function totalPrice() {
+  var sum = 0;
+var collection = document.getElementsByClassName("money");
+for (var i = 0; i < collection.length; i++) {
+  console.log(parseInt(collection[i].innerHTML));
+  sum = sum + parseInt(collection[i].innerHTML);
+}
+ console.log('SUM: ',sum);
+ document.querySelector("#totalPrice").innerHTML = sum;
 }
+totalPrice();
 
-function createNewItemRow(itemName, itemUnitPrice){
 
-}
 
-function createNewItem(){
+function removeItem() {
```

Comment:
Good job.  The page works reasonably well but your 'add more' button does not work.  
Look at the solution code I posted in the google drive to compare/contrast